apps/src/templates/studioHomepages/TopCourse.jsx

```diff
@@ -74,14 +74,14 @@ const styles = {
 // While this is named TopCourse, it really refers to the most recent course
 // or script in which the student or teacher has progress.
 
-const TopCourse = React.createClass({
-  propTypes: {
+class TopCourse extends Component {
```

Comment:
So, you can do your export and Radium-wrap inline here, and I'd favor that for a couple of reasons:
- I like seeing the export keyword right next to the exported function/object/whatever.
- We're never going to use this class without its Radium-wrapper.
It looks like this:
```jsx
export default Radium(class TopCourse extends Component {
  // ... class definition
});
```
Of course, that can also get to looking pretty ridiculous if you add more wrappers.  I'm not sure how other devs will feel about it.  Consider this comment nonblocking.